docs/src/references/agent/operation-workflow.md

```diff
@@ -164,8 +164,12 @@ on_error = "failed"
 
 [init]
   script = "/usr/bin/firmware_handler.sh plan"
-  on_success = "install"
+  on_success = "executing"
   on_error = { status = "failed", reason = "not timely" }
+  
+[executing]
+  action = "proceed"
+  on_success = "install"
```

Comment:
This is unrelated - but as the executing step was missing the example was not working with Cumulocity.